roles/elk/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: install oauth2_proxy
+  command: '/usr/local/bin/bash -c "export GOROOT=/usr/local/go GOPATH=/usr/local; go get github.com/bitly/oauth2_proxy"'
```

Comment:
ah. I wasn't aware of the shell module..

indeed, I dont use environment because of the proxy